lib/sparse/src/index/inverted_index/mod.rs

```diff
@@ -15,4 +20,28 @@ pub trait InvertedIndex {
 
     /// The number of indexed vectors, currently accessible
     fn indexed_vector_count(&self) -> usize;
+
+    /// Upsert a vector into the inverted index.
+    fn upsert(&mut self, id: PointOffsetType, vector: SparseVector);
+
+    /// Create inverted index from ram index
+    fn from_ram_index<P: AsRef<Path>>(
+        ram_index: InvertedIndexRam,
+        path: P,
+    ) -> std::io::Result<Self>
+    where
+        Self: Sized;
+
+    /// Returns the maximum number of results that can be returned by the index for a given sparse vector
+    fn max_result_count(&self, query_vector: &SparseVector) -> usize {
```

Comment:
I propose to move this function out of this trait. Because this function is required for test only and not required in vector index

Reply:
You are right, it does not belong in this trait :+1: 
I decided to move in in the sparse index for simplicity https://github.com/qdrant/qdrant/pull/2900/commits/50757440162bd15b006e15c0c9592de4871154c9